src/Helper/Product/Stock/Utility.php

```diff
@@ -119,10 +104,17 @@ public function getStockItem($product): StockItem
     {
         /** @var StockItem $shopgateStockItem */
         $shopgateStockItem = $this->shopgateStockItemFactory->create();
-        if (version_compare($this->getCurrentVersion(), '2.3.0', '>=')) {
-            $stockId         = $this->getStockIdForCurrentWebsite->execute();
-            $stockItemData   = $this->getStockItemData->execute($product->getSku(), $stockId);
-            $stockItemConfig = $this->getStockItemConfiguration->execute($product->getSku());
+        if (version_compare($this->getCurrentVersion(), '2.3.1', '>=')) {
```

Comment:
How would that work when the claim from the ticket is right, that e.g. GetStockItemConfiguration does not exist in 2.3.1?

Reply:
oh wow, I wonder if the tests I asked Alex to create in that PR never actually test the stock ^_^ And the only reason it works is because I moved it out of constructor

Reply:
yup, assumption was correct. It was me moving it out of the constructor that created a false positive. There was another test class that was checking stock. Unfortunately automated tests would have caught my mistake. Glad Stephan eye test failed :D